openstates/metadata/data/legacy_districts.py

```diff
@@ -1,6 +1,22 @@
 from ..models import District
 
 legacy_districts = {
```

Comment:
Worth leaving a comment here explaining what `legacy_districts` are (now that we know...)?

Reply:
Agreed

Reply:
I have added a comment explaining the use of legacy districts